samples/logging/log_collection_sample.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python
+
+# -*- coding: utf-8 -*-
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+#
+# Modifications Copyright OpenSearch Contributors. See
+# GitHub history for details.
+
+import logging
+import queue
+from datetime import datetime
+from logging.handlers import QueueHandler, QueueListener
+from typing import Any
+
+import urllib3
+
+from opensearchpy import OpenSearch
+
+urllib3.disable_warnings()
+
+
+def main() -> None:
+    print("Running Log Collection Guide")
```

Comment:
Nit: remove the word guide, it's not a guide it's a sample maybe just say "Collecting logs" or something like that.